modules/apps/asset/asset-publisher-test/src/testIntegration/java/com/liferay/asset/publisher/exportimport/test/AssetPublisherExportImportTest.java

```diff
@@ -1233,9 +1232,6 @@ protected void testSortByAssetVocabulary(boolean globalVocabulary)
 	private ExportImportConfigurationLocalService
 		_exportImportConfigurationLocalService;
 
-	@Inject(filter = "model.class.name=com.liferay.portal.kernel.model.Layout")
```

Comment:
Hey @GabrielHenriquedev, break this commit in two. One for the deletion of the ExportImportLocalService/ExportImportService and another to ExportImportController.